src/cloud-functions.ts

```diff
@@ -86,6 +86,9 @@ function _makeParams (event: Event<any>, triggerResource: string): { [option: st
       params[wildcardNoBraces] = eventResourceParts[position];
     });
   }
+
+  Object.assign(params, event.params);
```

Comment:
The general principle I'd like to follow is that event.resource is the source of truth. But if that is absent (as in the case of some unit tests), then event.params should be used. I'm not sure how I feel about merging.

Could you do something to the beginning of the function along the lines of:
```
if (!triggerResource) {
   return event.params || {};
}
```